src/foundry/client/pixi/sources/light-source.d.ts

```diff
@@ -1,12 +1,20 @@
+// FOUNDRY_VERSION: 10.291
+
 import type { ConfiguredObjectClassForName } from "../../../../types/helperTypes";
```

Comment:
Linter complaint.

Reply:
If I remove the import, I get this error:

> Augmentations for the global scope can only be directly nested in external modules or ambient module declarations.ts(2669)

I believe this is happening because of the `src\foundry\client\pixi\index.d.ts` importing this file.

Can I go ahead and put the following in this file to fix the TS error?

```ts
export {};
```

Reply:
In that case, yes. Add it to the start of the file where the imports would usually be. That way we don't forget to remove it later.

Reply:
Updated